.github/workflows/continuous-integration.yml

```
@@ -5,90 +5,51 @@
         with:
           php-version: ${{ matrix.php-version }}
-          coverage: xdebug
-          ini-values: zend.assertions=1
+          coverage: none
+          tools: none
+          ini-values: assert.exception=1, zend.assertions=1
 
       - name: Get composer cache directory
         id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)